bot.py

```diff
@@ -81,11 +81,11 @@ def on_message(mosq, obj, msg):
         end_to_bot("Unknown card at door", increment = True)
```

Comment:
could "end_to_bot" be causing it to fall over?

Reply:
Very likely! This is fixed in master and merged back into this PR :)